File: LeetCode/35. 搜索插入位置.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var searchInsert = function (nums, target) {
  let notFound = 0
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] === target) {
      return i
    } else if (nums[i] < target) {
      notFound++
    }
  }
  return notFound
}

/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var searchInsert2 = function (nums, target) {
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] >= target) {
      return i
    }
  }
  return nums.length
}

File: LeetCode/1-100/35. 搜索插入位置.java
class Solution {
    public int searchInsert(int[] nums, int target) {
 int n = nums.length;
        for (int i1 = 0; i1 < nums.length; i1++) {
            if (nums[i1] >= target) return i1;
        }
        return n;
    }
        public int searchInsert(int[] nums, int target) {
    for (int i = 0; i < nums.length; i++) {
                if (nums[i] >= target) return  i;
        }
        return nums.length;
    }
}